### ISSUE TITLE
Target lost after attack

### ISSUE THREAD

**Issue by Violettdragon** [Plushtoast/dsa5-foundryVTT#157]
Hello Again. :)

It is possible to let the target on the last chosen token, after an attack?

Is it also possible that the token saves the last target and the GM does not have to activate his target every time?

**Comment by Plushtoast** [comment]
This is a intended feature at the moment. GM's accidentally target multiple tokens when the target is not cleared automatically.
Most fights there is not only one target for the GM.

**Comment by mckee14** [comment]
I agree - you get used to it very quick, and the other way around can be a pitfall.

Try the module "next up!", it's a fantastic helper for combat: automatically pans to the active token, opening his charsheet, and closes it after the turn, with options to suit your style.
With this module enabled, you can switch to the "Target" tool (left hand side, where you have all your tools). Now, you can just click the needed target(s), klick the sheet-button, klick next. 

Another nice module is "combat focus", you can see the chat window below the turn tracker, so you don't have to swich.

Reduced the time per token from about 15 seconds to 3 seconds in a fight.

**Comment by Violettdragon** [comment]
"next up!" is a nice tool, thanks for the hint.
I dont need "Combat Focus" , i open the extra window with right mouse click on it. But thanks anyway.

**Comment by Plushtoast** [comment]
I will add a setting to override this behavior
